README.md

```diff
@@ -63,7 +63,7 @@ All the built files are generated in the `dist` directory.
 - Go to `chrome://extensions`.
 - Turn ON `Developer mode`.
 - Click on `Load Unpacked`.
-- Select the `manifest.json` file in `dist` folder that was generated after the build step. The extension should now show up on the list.
+- The `manifest.json` file that was generated after the build step, should be uploaded. Select the `dist` folder after clicking on `Load Unpacked` and it will check for manifest on its own. The extension should now show up on the list.
```

Comment:
Can you remove the first sentence
> The `manifest.json` file that was generated after the build step, should be uploaded.

Remove "after clicking on `Load Unpacked`", as it's already defined in the previous step.
Change "for manifest on..."   to "for `manifest.json` on..."

Reply:
@blenderskool Done :+1: